code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -253,7 +248,7 @@
 
 /datum/reagent/medicine/oxandrolone/on_mob_life(mob/living/M)
 	if(M.getFireLoss() > 50)
-		M.adjustFireLoss(-4*REM, 0) //Twice as effective as silver sulfadiazine for severe burns
+		M.adjustFireLoss(-2*REM, 0)
```

Comment:
It is weird to make something objectively worse than silver sulfadiazine, it's just a trap for people who don't know better.

Reply:
'it's weird to make a chemical worse than silver sulfadizine at healing inside the body after silver sulfadizine no longer metabolizes over time to heal you when this easily produced chemical is the single best method for treating burns above all other chemicals and would otherwise tip balance horribly as it still does'

k